content/docs/configuration.md

```diff
@@ -17,70 +16,78 @@ You should place the file in the root of your upstream repo. Packit accepts thes
 * `packit.yml`
 * `packit.json`
 
+Both Packit-as-a-Service and packit tool use this configuration file.
 
-### Values
+
+## Top level keys
 
  Key name                  | Type            | Description
 ---------------------------|-----------------|----------------------------------------------------------------------
  `specfile_path`           | string          | relative path to a spec file within the upstream repository (mandatory)
- `synced_files`            | list of strings | a list of relative paths to files in the upstream repo which are meant to be copied to dist-git during an update
- `upstream_project_name`   | string          | name of the upstream repository (e.g. in PyPI); this is used in `%prep` section
+ `upstream_project_name`   | string          | name of the upstream repository (e.g. in PyPI); this is used in `%prep` section to generate an archive
+ `create_pr`               | bool            | when doing a new update in Fedora dist-git, should packit create a new pull request (when set to `true`) or push directly to dist-git (defaults to `false`)
```

Comment:
This is not right, default value is True, means create a new pull request.
User should allow to directly push into dist-git. https://github.com/packit-service/packit/blob/master/packit/config.py#L361

Reply:
I have selected this in  PR https://github.com/packit-service/packit/pull/416

Reply:
Sorry I did not see this PR https://github.com/packit-service/packit/pull/419